.eslintrc.json

```
@@ -127,10 +127,10 @@
         "yield": "yields"
       },
       "preferType": {
-        "Array": "array",
+        "Array": "Array",
```

```suggestion
        "array": "Array",
```

See https://eslint.org/docs/rules/valid-jsdoc#prefer.